### PR TITLE
OPENEUROPA-1538: Adjust Behat tests for OE Webtools Analytics rules.

### PR DESCRIPTION
**Pull request by sergepavle**
### Description

Add Behat scenario for adding siteSection for checking current page alias by regular expression without checkbox.
Add context to clean up created during scenario executions configs and db records.

### CODE REVIEW

File: tests/features/multilingual-aliases.feature [hunk 1]
```diff
@@ -23,16 +24,47 @@ Feature: Webtools Analytics multilingual aliases
...
     Then the page analytics json should contain the parameter "siteSection" with the value "multilingual"
-    Given I am on "/nl/nieuws"
+    When I am on "/nl/nieuws"
     Then the page analytics json should contain the parameter "siteSection" with the value "multilingual"
-    Given I am on "/admin"
+    When I am on "/admin"
     Then the page analytics json should not contain the parameter "siteSection"
 
+  @cleanup:webtools_analytics_rule @cleanup-aliases
+  Scenario: Make sure that Webtools Analytics rules work without checked "Match on path alias for site default language"
```
Comment (upchuk) [inline tests/features/multilingual-aliases.feature]: `Make sure that Webtools Analytics rules work without matching the alias in the site default language`

File: tests/Behat/WebtoolsCleanupContext.php [hunk 2]
```diff
@@ -0,0 +1,143 @@
...
+   * @param \Behat\Behat\Hook\Scope\BeforeScenarioScope $scope
+   *   The scenario scope.
+   *
+   * @beforeScenario @cleanup-aliases
+   */
+  public function collectExistingUrlAliases(BeforeScenarioScope $scope): void {
+    // Reset the alias list at the beginning of each scenario.
+    $this->existingAliases = [];
+    // Executing database query,
+    // because AliasStorage->load() returns only a single url alias.
```
Comment (upchuk) [inline tests/Behat/WebtoolsCleanupContext.php]: Comment can go in one line.

File: tests/features/multilingual-aliases.feature (hunk 1 above)
Comment (sergepavle) [reply tests/features/multilingual-aliases.feature]: Done

File: tests/Behat/WebtoolsCleanupContext.php (hunk 2 above)
Comment (sergepavle) [reply tests/Behat/WebtoolsCleanupContext.php]: I did rephrase. Initially I used second line for avoiding code sniffer error.

**Pull request merged**